slice cases from top chunk (#145)
epiverse-trace/tutorials-middle: episodes/quantify-transmissibility.Rmd

> There are case data available for `r dim(cases)[1]` days, but in an outbreak situation it is likely we would only have access to the beginning of this data set. Therefore we assume we only have the first 90 days of this data. 

Author: ```suggestion
In an outbreak situation it is likely we would only have access to the beginning of the input data set. Therefore we assume we only have the first 90 days of this data. 
```
